.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,23 @@
+name: Docker Image CI
+
+on: [push]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
```

Comment:
I think this action can be removed because the job isn't using anything from the repo